Resources/ServerInfo/Guidebook/_RMC14/Law/RMCMarineLawArresting.xml

```diff
@@ -0,0 +1,43 @@
+﻿<Document>
+
+  # Arresting Procedures
+  Should the warship get boarded by a hostile force no arrests should be made unless the person is a danger to the warship and or its personnel.
+
+  ## Officer Ordered Arrests
+  Should an Officer ranked UNMC personnel order an arrest for a crime, MPs MUST arrest that person
+  - Only the [color=#008000][bold]Commander[/bold][/color] [italic](not a [color=#90EE90][bold]Duty Officer[/bold][/color])[/italic] and [color=#8B0000][bold]Chief MP[/bold][/color] can Officer order an arrest on an MP.
+  - They may arrest and hold that person for ten minutes after they have reached the brig while they gather evidence.
+  - Should the suspect be declared innocent of all accused crimes, the requesting officer may incur a Prevarication charge. The time for any crimes the suspect committed as a result of the arrest must still be served, however.
+  - Before the suspect is detained, the Officer may recind the order.
+  - In the case of [bold]officer ordered arrests[/bold] the [color=#8B0000][bold]Chief MP[/bold][/color] or [color=#008000][bold]Commander[/bold][/color] may choose to assign an NJP over the initial charge (where valid).
+
+  ## Arresting a Suspect
+  [italic]These arrests can be performed anywhere on the warship.[/italic]
+  - Take down the Suspect if they resist and securely restrain them.
+  - Move the Suspect to a brig cell.
+  - Continue to [bold]charging procedure[/bold].
```

Comment:
When "Take down the suspect" is put into procedure, this encourages a potentially problematic zero-conversation approach. I recommend adding a requirement to inform the suspect they are being arrested as a first step to avoid this.

Reply:
This may be updated in the future but is currently required for our playerbase